app/mailers/chomp_session_mailer.rb

```diff
@@ -0,0 +1,16 @@
+class ChompSessionMailer < ApplicationMailer
+
+  # Subject can be set in your I18n file at config/locales/en.yml
+  # with the following lookup:
+  #
+  #   en.chomp_session_mailer.create_chomp.subject
+  #
+  def create_chomp
+    @chomp_session = params[:chomp_session]
+
+    mail(
+      to:       @chomp_session.user.email,
```

Comment:
is there a check if the user is a guest but did not specify his email?

Reply:
The user who created the chomp session needs an account so an email will already be there

Reply:
okay 🤩